test/e2e/tests/send-eth.spec.js

```diff
@@ -91,6 +91,8 @@ describe('Send ETH from inside MetaMask using default gas', function () {
   });
 });
 
+// TODO wrap all of this in a describe
+/* eslint-disable mocha/max-top-level-suites */
```

Comment:
With the updates to the ESLint config, these test files are now failing the lint step. I suspect this is the case because not all of the Mocha rules were getting applied to the e2e tests previously and now they are.